libs/motr/motr_core_k8s_lib.py

```diff
@@ -348,6 +350,27 @@ def md5sum_cmd(self, file1, file2, node):
         assert_utils.assert_not_in("ERROR" or "Error", resp,
                                    f'"{cmd}" Failed, Please check the log')
 
+    def get_md5sum(self, file, node):
```

Comment:
This function is as good as issuing a command into a Cortx container. If we anyway want to have that , can you add a suffix (just for ref hax_container suffix) to function name ?

Reply:
All motr utilities run from hax container, so I have added the description to this function mentioning the same. The same applies to dd, m0cat, m0cp and diff functions.